app/services/idv/actions/ipp/redo_address_action.rb

```diff
@@ -0,0 +1,11 @@
+module Idv
+  module Actions
```

Comment:
Yeah it's kind of weird but I'm just going with the existing pattern that's been used since we started. I think eventually it'd be worth revisiting, probably as part of [LG-6663](https://cm-jira.usa.gov/browse/LG-6663)

I think `Idv::InPerson::Actions::RedoAddressAction` is somewhat better but I prioritized consistency in this PR since we use a similar pattern for steps, eg [`Idv::Steps::Ipp::SsnStep`](https://github.com/18F/identity-idp/blob/main/app/services/idv/steps/ipp/ssn_step.rb)

Reply:
thank you! I wasn't sure how nested modules worked. I'll have to take a look and look at the various patterns we're using and get used to them.